src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -266,6 +266,15 @@
     <Compile Include="Implementation\Diagnostics\DiagnosticsSuggestionTaggerProvider.cs" />
     <Compile Include="Implementation\Diagnostics\SuggestionAdornmentManagerProvider.cs" />
     <Compile Include="Implementation\Diagnostics\SuggestionTag.cs" />
+    <Compile Include="Implementation\FindReferences\DefinitionItem.cs" />
```

Comment:
No.  That's totally valid.  These can be at the workspace level as they're pure data.